### DISCUSSION TITLE
DataHub configuration can't be saved (not writable anymore)

### DISCUSSION THREAD

**Discussion by kilbeggan** [pimcore/pimcore#14114] [Q&A / Support] [answered]
Hi. Suddenly the DataHub configuration can't be saved and can't understand the reason. I can't even add a new one. There are no suspicious errors in the log. Any comments and suggestions will be appreciated.

![image](https://user-images.githubusercontent.com/11678128/213689902-f99ae38f-2093-43a6-92c4-b5c29ea15c2f.png)

![image](https://user-images.githubusercontent.com/11678128/213690070-3922fee4-edbe-4783-acd0-0863beed1145.png)

**Comment by mcop1** [comment, marked as answer]
@kilbeggan please have a look at the docs here https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Configuration_Environments.html

**Reply by agdrs** [reply]
Hi @kilbeggan 

i do not understand, why APP_DEBUG configuration is a dependency here.
Locally, i can set it to TRUE, ok. But in staging, i set it to FALSE and then no configuration can be added or changed, if done with symfony-config.
Why is handled that way? Why does it not depend on roles and/or permissions.

**Reply by kilbeggan** [reply]
@agdrs I guess this question is not for me.

**Reply by endeehjeeh** [reply]
@kilbeggan 
sorry, no, thanks for the hint...
I meant @mcop1

**Comment by kilbeggan** [comment]
Thank you, rtfm is working well. The solution is .env, APP_DEBUG=true.
